README.md

```diff
@@ -3,16 +3,82 @@
 [![][travis img]][travis]
 [![][license img]][license]
 
-#Knot.x - reactive multisource assembler 
+#Knot.x - efficient, high-performance and scalable integration platform for modern websites   
 
 <p align="center">
-  <img src="https://github.com/Cognifide/knotx/blob/master/icons/180x180.png?raw=true"
-         alt="Knot.x"/>
+  <img src="https://github.com/Cognifide/knotx/blob/master/icons/180x180.png?raw=true" alt="Knot.x Logo"/>
 </p>
 
 
 ## What is Knot.x?
-Very efficient, high-performance and scalable platform which assembles static and dynamic content from multiple sources.
+Let's imagine **an online banking website** containing different features like a *chat box*, *exchange rates*, 
+*stock data* and *user profile information*. The site has a high performance characteristic, despite of 
+its complexity and *target publishing channels*.
+
+All those features come from different providers/vendors having their own teams working in various modes, 
+technologies and release cycles.
+
+Knot.x connects all of them in a controlled and isolated way, preventing any undesired interferences.
+It combines **asynchronous programming principles** and **message-driven architecture** providing **a scalable 
+platform** for modern sites.
+
+**Knot.x** connects all of the above **in a unified customer experience**.
+
+
+## What problems does Knot.x solve?
+
+###Features
+
+<img align="right" 
+  src="https://github.com/Cognifide/knotx/blob/master/documentation/src/main/wiki/assets/knotx-intro-features.png?raw=true"
+  alt="Features"/>
+
+Probably you have many features / services you want to connect to your site. They come from 
+different vendors, talk using various protocols (REST / SOAP / binary) and have different 
```

Comment:
'talk using various protocols (REST / SOAP / binary)'
into
'talk using various network protocols (HTTP/WebSocket/TCP etc.) and with message format (SOAP/JSON/XML/binary etc.)'